funding/manager.go

```diff
@@ -4512,3 +4485,24 @@ func (f *Manager) selectShutdownScript(taprootOK bool,
 
 	return txscript.PayToAddrScript(addr)
 }
+
+// waitForPeerOnline blocks until the peer specified by peerPubkey comes online
+// and then returns the online peer.
+func (f *Manager) waitForPeerOnline(peerPubkey *btcec.PublicKey) (lnpeer.Peer,
```

Comment:
Is there an easy way to add a unit test for this new method?

Reply:
hmm what's NFC? On the one hand, we do have tests to cover it, on the other hand, they need to be improved, for simplicity and maintainability. And we try to patch tests whenever possible.

Reply:
NFC == "No functional change". Means this is a pure refactoring/cleanup.

If we need a new test here, I may be able to get around to it in a week or two, but it's not high priority for me ATM.  (Currently focused on https://github.com/lightningnetwork/lnd/issues/6569).